politeiawww/user/cockroachdb/cms.go

```diff
@@ -267,6 +267,42 @@ func (c *cockroachdb) cmdCMSUserByID(payload string) (string, error) {
 	return string(reply), nil
 }
 
+func (c *cockroachdb) cmdCMSUserSubContractors(payload string) (string, error) {
+	// Decode payload
+	p, err := user.DecodeCMSUserByID([]byte(payload))
+	if err != nil {
+		return "", err
+	}
+	var cmsUsers []CMSUser
+	err = c.userDB.
+		Where("supervisor_user_id = ?", p.ID).
```

Comment:
What happens when a contractor does work for multiple corporate contractors?

Reply:
Hrm I'll discuss with @behindtext but I believe that won't be allowed at least in this first iteration.

I believe there's an implicit idea that sub contractors will only work directly with 1 corporate contractor.

Reply:
Also after reviewing the code I realized that's it's impossible to currently set multiple supervisor user id's.  It's 1 or none.

Reply:
So lets say there is a marketing corporate contractor in the future.  It's not uncommon for a dev, who would probably already be classified under a dev corporate contractor, to then do some work (podcasts, events, etc) that would fall under the marketing corporate contractor's budget.  How would the marketing corporate contractor bill for this if they can't attribute it to the dev that did the work?

Reply:
I bring this up because it's going to be a huge headache if we architect cms with this assumption and it needs to be changed in the future.

I could also see devs working on different projects that fall under different corporate contractor's budgets.